### PR TITLE
added request and json_request helpers - fixes #50

### LINKED ISSUE #50: create synchronous version of HTTP calls
https://github.com/mozilla-services/screenshots-loadtests/blob/master/utils.py#L113

so people don't have to do all that

### PR DESCRIPTION
**Pull request by tarekziade**
- [X] Add synchronous helpers
- [x] Make sure rpapa likes the API
- [x] Make sure  pdehaan likes the API
- [x] Add an example in the documentation
- [x] profit

### CODE REVIEW

File: molotov/tests/test_fmwk.py [hunk 1]
```diff
@@ -115,10 +123,16 @@ def test_runner_multiprocess(self):
...
         args.processes = 2
         args.workers = 5
+        args.console = console
         results = runner(args)
         self.assertTrue(results['OK'] > 0)
         self.assertEqual(results['FAILED'], 0)
 
+    @dedicatedloop
+    def test_runner_multiprocess_console(self):
+        self._multiprocess(console=True)
```
Comment (rpappalax) [inline molotov/tests/test_fmwk.py]: does console=True serve as a flag to output stdout?

File: molotov/util.py [hunk 2]
```diff
@@ -121,3 +124,45 @@ def expand_options(config, scenario, args):
...
+
+
+def _run_in_fresh_loop(coro):
+    loop = asyncio.new_event_loop()
+    res = None
+    try:
+        task = loop.create_task(coro(loop=loop))
+        res = loop.run_until_complete(task)
+    finally:
+        loop.close()
```
Comment (rpappalax) [inline molotov/util.py]: looks like this allows the loop to be closed cleanly (and prevents the errors we were seeing before)

**Comment by rpappalax** [comment]
looks like request and json_request provide a clean way to create and manage individual loop(s) to essentially allow for activity to take place synchronously.  looks good!

File: molotov/tests/test_fmwk.py (hunk 1 above)
Comment (tarekziade) [inline molotov/tests/test_fmwk.py]: console is the flag that determine if Molotov runs with the console or the curses output (-c in the shell)

**Comment by tarekziade** [comment]
Thanks @rpappalax ! I will wait for @pdehaan feedback and then add the doc

**Pull request merged**